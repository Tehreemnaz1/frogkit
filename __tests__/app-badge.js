import { mount } from 'vue-test-utils'
import Vue from 'vue'
import AppBadge from '@/components/app-badge'

const appStoreItems = {
  buttons: [
    {
      name: 'apple',
      url: 'https://app.adjust.com/p39urb',
      label: 'Laden im'
    },
    {
      name: 'google',
      url: 'https://app.adjust.com/vti0vj',
      label: 'JETZT BEI'
    }
  ]
}

describe('App Badge component', () => {
  it('Should render component and match snapshot', () => {
    const wrapper = mount(AppBadge, { propsData: appStoreItems })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
