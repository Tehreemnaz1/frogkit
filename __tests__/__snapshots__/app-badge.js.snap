// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Badge component Should render component and match snapshot 1`] = `<a href="https://app.adjust.com/p39urb" target="_blank" rel="noopener" class="fk-app-btn apple">Laden im</a>`;
